frontend/src/properties/propertysections/WidgetContainerPropertySection.tsx

```diff
@@ -0,0 +1,149 @@
+import { PropertySectionLabel } from 'properties/propertysections/PropertySectionLabel';
+import {
+  GQLErrorPayload,
+  GQLUpdateWidgetFocusInput,
+  GQLUpdateWidgetFocusMutationData,
+  GQLUpdateWidgetFocusMutationVariables,
+  GQLUpdateWidgetFocusPayload,
+  WidgetContainerPropertySectionProps,
+} from 'properties/propertysections/WidgetContainerPropertySection.types';
+import React, { useEffect, useState } from 'react';
+import { v4 as uuid } from 'uuid';
+
+export const updateWidgetFocusMutation = gql`
+  mutation updateWidgetFocus($input: UpdateWidgetFocusInput!) {
+    updateWidgetFocus(input: $input) {
+      __typename
+      ... on ErrorPayload {
+        message
+      }
+    }
+  }
+`;
+
+const useWidgetContainerPropertySectionStyles = makeStyles(() => ({
+  root: {
+    flexGrow: 1,
+  },
+}));
+
+const isErrorPayload = (payload: GQLUpdateWidgetFocusPayload): payload is GQLErrorPayload =>
+  payload.__typename === 'ErrorPayload';
+
+const getLayout = (layout: WidgetContainerLayout): GridDirection => {
+  if (layout === WidgetContainerLayout.HORIZONTAL) {
+    return 'row';
+  }
+  return 'column';
+};
+
+export const WidgetContainerPropertySection = ({
+  editingContextId,
+  formId,
+  widget,
+  widgetSubscriptions,
+  setSelection,
+  readOnly,
+}: WidgetContainerPropertySectionProps) => {
+  const classes = useWidgetContainerPropertySectionStyles();
+
+  const [message, setMessage] = useState<string | null>(null);
+
+  const [
+    updateWidgetFocus,
+    { loading: updateWidgetFocusLoading, data: updateWidgetFocusData, error: updateWidgetFocusError },
+  ] = useMutation<GQLUpdateWidgetFocusMutationData>(updateWidgetFocusMutation);
+  const sendUpdateWidgetFocus = (selected: boolean) => {
+    const input: GQLUpdateWidgetFocusInput = {
+      id: uuid(),
+      editingContextId,
+      representationId: formId,
+      widgetId: widget.id,
+      selected,
+    };
+    const variables: GQLUpdateWidgetFocusMutationVariables = {
+      input,
+    };
+    updateWidgetFocus({ variables });
+  };
+
+  useEffect(() => {
+    if (!updateWidgetFocusLoading) {
+      if (updateWidgetFocusError) {
+        setMessage('An unexpected error has occurred, please refresh the page');
+      }
+      if (updateWidgetFocusData) {
+        const { updateWidgetFocus } = updateWidgetFocusData;
+        if (isErrorPayload(updateWidgetFocus)) {
+          const { message } = updateWidgetFocus;
+          setMessage(message);
+        }
+      }
+    }
+  }, [updateWidgetFocusLoading, updateWidgetFocusData, updateWidgetFocusError]);
+
+  const onFocus = () => sendUpdateWidgetFocus(true);
+  const onBlur = () => sendUpdateWidgetFocus(false);
+  const layout = getLayout(widget.layout);
+
+  return (
+    <div>
+      <PropertySectionLabel label={widget.label} subscribers={[]} />
+      <Grid
+        className={classes.root}
+        data-testid={widget.label}
+        container
+        onFocus={onFocus}
+        onBlur={onBlur}
```

Comment:
Are the `onFocus` and `onBlur` pertinent for the grid?

Reply:
No you are right, the WidgetContainerPropertySection has no need to be focusable on the Form representation.